### PR TITLE
[#444] Fix: Deploy Appstore Workflow failed cause of missing environment variables

### LINKED ISSUE #444: [Bug] Fix: Deploy Appstore Workflow failed cause of missing environment variables.
## Issue

- The `test_upload_build_to_test_flight` workflow run successfully and it inject the environment variables while calling fastlane buildAndUploadToAppStore but the `deploy_app_store` workflow didn't. We need to update deploy_app_store to inject the environment variables.

Screenshot 2023-02-15 at 15 16 40Screenshot 2023-02-15 at 15 20 17

- The `FIREBASE_CLI_TOKEN` variable is the same with GitHubAction `FIREBASE_TOKEN`. We should rename it to be the same to avoid confusion.
 
## Expected

- Update `deploy_app_store` to inject the environment variables 
- Rename `FIREBASE_CLI_TOKEN` to `FIREBASE_TOKEN`
- Deploy Appstore workflow run successfully
 
## Steps to reproduce

1. Use the template on a new repository
2. Run make.sh
3. Configure Constant.swift variables
4. Add GitHubAction Secrets as Wiki Document describe
5. Run workflow Deploy Appstore and it will be failed

### PR DESCRIPTION
**Pull request by ducbm051291**
- Closes #444 

## What happened

- Update Deploy Appstore Workflow: Add env variables in step Build and Upload to Appstore
```
API_KEY_ID: ${{ secrets.API_KEY_ID }}
ISSUER_ID: ${{ secrets.ISSUER_ID }}
BUMP_APP_STORE_BUILD_NUMBER: "true"
```
- Update Secret.swift key name: Rename `FIREBASE_CLI_TOKEN` to `FIREBASE_TOKEN`
 
## Insight

How I tested:
- Created branch `bug/444-test` to store a setup project.
- Run Github Action `Deploy Build To App Store` on that branch.
- Observe the result.
 
## Proof Of Work

CleanShot 2023-04-26 at 11 36 11@2x

### CODE REVIEW

**Review by vnntsu — APPROVED** [review]

File: .github/workflows/deploy_app_store.yml [hunk 1]
```diff
@@ -4,6 +4,9 @@ name: Deploy Build To App Store
 ### SSH_PRIVATE_KEY for Match Repo
 ### MATCH_PASS
 ### APPSTORE_CONNECT_API_KEY
+### API_KEY_ID
+### ISSUER_ID
+### APPSTORE_CONNECT_API_KEY
```
Comment (blyscuit) [inline .github/workflows/deploy_app_store.yml:9]: Duplicated line with line 6.
```suggestion
```

File: fastlane/Managers/EnvironmentManager.rb [hunk 2]
```diff
@@ -1,26 +0,0 @@
-# frozen_string_literal: true
-
-class EnvironmentManager
```
Comment (blyscuit) [inline fastlane/Managers/EnvironmentManager.rb:3]: I think this change should be another PR referencing https://github.com/nimblehq/ios-templates/issues/330 instead, because removing a helper class is a quite big change 🤔.

**Review by blyscuit — COMMENTED** [review]
Fix looks good.

File: fastlane/Managers/EnvironmentManager.rb (hunk 2 above)
Comment (ducbm051291) [reply fastlane/Managers/EnvironmentManager.rb:3]: This class has been unused since we moved to Fastlane Swift so I think we can clean it up in this PR :D

Comment (blyscuit) [reply fastlane/Managers/EnvironmentManager.rb:3]: Thank you for the clarification 🙏

**Review by blyscuit — APPROVED** [review]

**Review by suho — APPROVED** [review]

**Review by markgravity — APPROVED** [review]
